targets/TARGET_STM/TARGET_STM32F3/TARGET_STM32F302x8/TARGET_NUCLEO_F302R8/PeripheralPins.c

```diff
@@ -226,13 +226,13 @@ const PinMap PinMap_SPI_SSEL[] = {
 };
 
 const PinMap PinMap_CAN_RD[] = {
-//  {PB_8 , CAN_1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_NOPULL, GPIO_AF9_CAN)}, // Not available in 32 pins package
+    {PB_8 , CAN_1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_NOPULL, GPIO_AF9_CAN)},
```

Comment:
This comment was a mistake. The package used on this board is a 64pin and not a 32pin. So these pins are available.